ucs/clientlist.cpp

```diff
@@ -793,7 +793,10 @@ void Clientlist::ProcessOPMailCommand(Client *c, std::string command_string, boo
 	case CommandJoin:
 		if (!command_directed) {
 			//Append saved channels to params
-			parameters = parameters + ", " + database.CurrentPlayerChannels(c->GetName());
+			auto const savedChannels = database.CurrentPlayerChannels(c->GetName());
+			if (savedChannels.length() > 0) {
+				parameters += fmt::format(", {}", savedChannels);
+			}
```

Comment:
`const auto`, would name it `saved_channels` as well.

Reply:
Not that `auto const` and `const auto` aren't the same, it's just easier to read and fits with other places that do this. Variables are generally snake-cased.